pkg/cri/relay/relay.go

```diff
@@ -134,6 +150,34 @@ func (r *relay) Server() server.Server {
 	return r.server
 }
 
+func (r *relay) dialNotify(socket string, perm os.FileMode, uid int, gid int, err error) {
+	if err != nil {
+		r.Error("failed to determine permissions/ownership of client socket %q: %v",
+			socket, err)
+		return
+	}
+
+	// we ignore image socket properties, unless it is the only socket we have
```

Comment:
Maybe explain here why this design decision?

Reply:
Added a comment explaining what's going on and wy.